data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,38 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors bordering Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. Once you return to <planet> you will receive a logistics adjustment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Will Not Someone Please Think Of The Children: done"
+		not "Remnant: Disable 1: aborted"
+		not "Remnant: Disable 1: failed"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic target marked staying disables plunders
+		system 
+			near "Vaticanus" 1
```

Comment:
Just to confirm, this covers Vaticanus, Egeria, and Prosa?
Those systems and those systems only are where you want this to spawn?

Reply:
Basically it needs to spawn in that cluster of sectors but not in sector with the station. This was the easiest way I could think of.